Live demo

abdulrahmanshr75 opened the pull request:
Live demo for the website

abdulrahmanshr75 committed:
Added live demo

Live demo for the website

Cathella reviewed (changes requested):
## 🔴 Status: Changes Requested 🛑

### Hi @abdulrahmanshr75 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Project Highlights 💪🏻

✔️ No linter errors.
✔️ Used GitHub Flow correctly.
✔️ Documented your README in a professional way.
✔️ Deployed portfolio website using GitHub Pages.

## Required Changes ♻️

- [ ] It would nice if you updated the `navigation links` to link to their respective sections of the portfolio. This is important because dead-links make a website look unprofessional 👍🏻

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abdulrahmanshr75 committed:
Updated the navigation

updated the navigation links to link to their respective sections of the portfolio

rustam-tolipov reviewed (approved):
## Hi @abdulrahmanshr75   👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

![image](https://user-images.githubusercontent.com/69011963/133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7.png)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
